test/kubetest/rbac/cluster_roles.go

```diff
@@ -224,9 +224,18 @@ func CreateRoleBinding(name, namespace string) Role {
 }
 
 func DeleteAllRoles(clientset kubernetes.Interface) error {
-	(&ClusterRole{}).Delete(clientset, RoleNames["admin"])
-	(&ClusterRole{}).Delete(clientset, RoleNames["view"])
-	(&ClusterRoleBinding{}).Delete(clientset, RoleNames["binding"])
+	if err := (&ClusterRole{}).Delete(clientset, RoleNames["admin"]); err != nil {
+		logrus.Error("Unable to delete kubernetes admin roles, aborting", err)
```

Comment:
Can we move `logrus.Errorf` from lines 228, 232, 236 to
https://github.com/networkservicemesh/networkservicemesh/blob/master/test/kubetest/k8s.go#L444 ?

Reply:
This is part of the public API, I'd be more comfortable duplicating the error rather than just returning it.